classes/class-ep-debug-bar-query-output.php

```diff
@@ -125,11 +134,25 @@ public static function render_query( $query, $type = '' ) {
 					);
 					?>
 				</div>
-
-				<div class="ep-query-result">
-					<strong><?php esc_html_e( 'Query Result:', 'debug-bar-elasticpress' ); ?> <div class="query-result-toggle dashicons"></div></strong>
-					<pre class="query-results"><?php echo esc_html( stripslashes( wp_json_encode( json_decode( $result, true ), JSON_PRETTY_PRINT ) ) ); ?></pre>
-				</div>
+				<?php if ( ! empty( $error ) ) : ?>
+					<div class="ep-query-error-code ep-query-response-code">
+						<?php
+						echo wp_kses_post(
+							sprintf( '<strong>Error:</strong> %s', $error[0] )
+						);
+						?>
+					</div>
+					<div class="ep-query-error-code ep-query-response-code">
+						<?php
+						echo wp_kses_post(
+							/* translators: Debug bar elasticpress error message */
+							sprintf( __( '<strong>Recommended Solution:</strong> %s', 'debug-bar-elasticpress' ), $error[1] )
+						);
+						?>
+					</div>
+					<?php
+				endif;
+				?>
```

Comment:
Let's bring these two elements to the top of the output. Also, we need to add an i18n function around `'<strong>Error:</strong> %s'` adding a `/* translators... */` comment to it.